recipes/default.rb

```diff
@@ -24,7 +24,7 @@
 # WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 #
 
-include_recipe 'python'
+python_runtime 'beaver'
```

Comment:
Ah, seems like these attributes are enough for me. Then, let it be unchanged.
@chewi, thank you for clarification!